packages/compass-indexes/src/modules/regular-indexes.ts

```diff
@@ -1,10 +1,11 @@
 import { isEqual, pick } from 'lodash';
-import type { IndexDefinition } from 'mongodb-data-service';
+import type { IndexDefinition, DataService } from 'mongodb-data-service';
 import type { AnyAction } from 'redux';
 import {
   openToast,
   showConfirmation as showConfirmationModal,
 } from '@mongodb-js/compass-components';
+import { fetchShardingKeys } from '@mongodb-js/compass-crud';
```

Comment:
We should probably move this to data service. We already aggregate indexes state from multiple sources, might as well add another one there instead of exposing this method from crud. Maybe we can have both: an explicit method to fetch shard keys, and also adding this logic to indexes method so that this info is available on the index